### PR TITLE
Sign In screen

### PR DESCRIPTION
**Pull request by eduardourso**
#### This PR adds

Implementation of the new shining sign in screen. 

#### Screenshots
| Old Sign In Screen | New Sign In Screen |
| --- | --- |
| screen shot 2017-01-30 at 1 28 07 pm | screen shot 2017-01-30 at 1 27 25 pm |

### CODE REVIEW

**Comment by stefanhoth** [comment]
Does this also remove the pop-up?

File: web/src/components/SignIn/SignInStyle.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```
Comment (ouchadam) [inline web/src/components/SignIn/SignInStyle.js]: this import isn't needed 👻

File: web/src/components/SignIn/SignInPage.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const signInStyle = require('./SignInStyle');
+import { requestSignIn as signIn} from '../../firebase';
+
+class SignInPage extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        console.log(this.props);
```
Comment (jackSzm) [inline web/src/components/SignIn/SignInPage.js]: Is this needed ?

**Comment by blundell** [comment]
looks really nice 💯  purple

**Comment by ouchadam** [comment]
looking good! 

Just a comment about the styling _styling_ 😆 , I tried to create a convention for it by using `aphrodite` 

https://github.com/novoda/peepz/blob/master/web/src/components/Controls.js#L7

```js
import { StyleSheet } from 'aphrodite/no-important';

const styles = StyleSheet.create({
  container: {
    height: '100px',
    width: '100px'
  },
  logo: {
    width: '100%'
  }
});

export default styles;

...

import { styles } from 'buttionStyle.js';
import { css } from 'aphrodite/no-important';

...

<div className={css(styles.container)} />
```

How do you feel about having component style classes called `componentName.style.js`?

File: web/src/components/SignIn/SignInPage.js (hunk 2 above)
Comment (eduardourso) [inline web/src/components/SignIn/SignInPage.js]: nope, we don't. good catch! 👍

**Comment by eduardourso** [comment]
@ouchadam I quite like the idea, is there anything special that I have to do to create the .style file? or just add that to the file's extension?

**Pull request merged**